python/seldon_core/seldon_methods.py

```diff
@@ -53,7 +58,13 @@ def predict(
     else:
         if hasattr(user_model, "predict_raw"):
             try:
-                return user_model.predict_raw(request)
+                response = user_model.predict_raw(request)
```

Comment:
I was thinking, if this was to be run by the engine, would the metrics still work as they were, or would there be duplicate metrics given that they are aggregated here and potentially at the engine level?

Reply:
Metrics will be exposed twice to Prometheus if run with Engine but counters will show correct value.